Controller/Ajax/Media.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * TurnTo_SocialCommerce
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2016 TurnTo Networks, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace TurnTo\SocialCommerce\Controller\Ajax;
+
+use \Magento\Framework\Controller\ResultFactory;
+
+class Media extends \Magento\Swatches\Controller\Ajax\Media
+{
+    /**
+     * Get product media by fallback:
+     * 1stly by default attribute values
+     * 2ndly by getting base image from configurable product
+     *
+     * @return string
+     */
+    public function execute()
+    {
+        $productMedia = [];
```

Comment:
Check to see if "use child sku" config option is enabled. If it's not enabled, just return the `parent::execute()`